cypress/integration/document/document-spec.js

```diff
@@ -12,12 +12,15 @@ describe( 'Document View', () => {
 
   describe( 'document detail view', () => {
     beforeEach( () => {
-      cy.intercept( 'GET', '**/api/v1/?**&size=0' )
-          .as( 'getAggs' );
-      cy.intercept( 'GET', '**/api/v1/?**&sort=created_date_desc' )
-          .as( 'getComplaints' );
+      let request = '/?**&sort=created_date_desc';
```

Comment:
I've had issues where a / before a ? causes the request to fail in cypress.

Reply:
Fixed in https://github.com/cfpb/ccdb5-ui/pull/376/commits/9f5959eb3dc307a2f6963e9f76b9602ab8ddfc87